package.json

```diff
@@ -12,7 +12,7 @@
     "start": "./bin/polyserve",
     "init": "typings install",
     "build": "if [ ! -f typings/main.d.ts ]; then npm run init; fi; tsc",
-    "test": "gulp test"
+    "test": "gulp test; exit 0"
```

Comment:
That answer you linked to mentions that `npm test` "squelches" `ELIFECYCLE` errors, and recommends only using this in non-test scripts. Were you seeing something different?